src/components/plots/abstractPlot.ts

```diff
@@ -451,6 +451,39 @@ export module Plot {
 
       return d3.selectAll(allSelections);
     }
+
+    /**
+     * Retrieves the closest PlotData to the specified x/y point within a specified value
+     *
+     * @param {number} xValue The x value to compare against
+     * @param {number} yValue The y value to compare against
+     * @param {number} withinValue The maximum distance the closest selection can be to the point (default = Infinity)
+     * @returns {PlotData} The closest plot data to the point within a specified value.  nulls and null selection returned otherwise
+     */
+    public getClosestData(xValue: number, yValue: number, withinValue: number = Infinity): PlotData {
+      var queryPoint = {x: xValue, y: yValue};
+
+      var closestDatum: any = null;
+      var closestSelection = d3.select();
+      var closestPixelPoint: Point = null;
+      var closestPointDistance = withinValue;
+
+      this._getDrawersInOrder().forEach((drawer) => {
+        drawer._getRenderArea().selectAll(drawer._getSelector()).each(function (datum) {
+          var selection = d3.select(this);
+          var pixelPoint = drawer._getClosestPixelPoint(selection, queryPoint);
+          var pointDistance = Plottable._Util.Methods.pointDistance(pixelPoint, queryPoint);
+          if (pointDistance < closestPointDistance) {
+            closestDatum = datum;
+            closestPixelPoint = pixelPoint;
```

Comment:
Main reason why I have this is possibly for the neat visual effect of... let's place a circle at this pixelPoint that is really close to my cursor / similar effects